Nodejs/Product/Nodejs/TypingsAquisitionTool/bin/install_typings

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env node
+/**
+    Tool to aquire typings used for NTVS IntelliSense.
+
+    This is a simple wrapper around `typings-core`, with the main
+    difference being that missing packages do not stop other typings
+    from being aquired.
+*/
+"use strict";
+var typings = require('typings-core');
+var events = require('events');
+var minimist = require('minimist');
+
+/**
+ * Create a promise that runs one or more promises sequentially.
+ */
+var runAll = function (promises) {
+    return promises.reduce(function (p1, p2) {
+        return p1.then(function (_) {
```

Comment:
My scala is showing there. It's helpful to indicate that a parameter is ignored sometimes.